deep_qa/layers/attention/matrix_attention.py

```diff
@@ -1,10 +1,11 @@
 from copy import deepcopy
+from typing import Dict
 
 from keras import backend as K
 from overrides import overrides
 
 from ..masked_layer import MaskedLayer
-from ...common.params import Params
+from ...common.params import pop_choice_with_default
```

Comment:
This is slightly different though - if we do that, to use the first value of `choices` as the default parameters you would have to explicitly pass `None`. Is that what you want?